### PR TITLE
fix some markdown issues

### PR DESCRIPTION
**Pull request by cartersocha**
Cleaned up some of the markdown issues. Didn't remove all of them but covered most

### CODE REVIEW

**Review by reyang — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -39,5 +39,6 @@ Instances of abusive, harassing, or otherwise unacceptable behavior
 may be reported by opening an issue
 or contacting one or more of the project maintainers.
 
-This Code of Conduct is adapted from the [Contributor Covenant](http://contributor-covenant.org), version 1.2.0,
-available at [http://contributor-covenant.org/version/1/2/0/](http://contributor-covenant.org/version/1/2/0/)
+This Code of Conduct is adapted from the [Contributor
```
Comment (reyang) [inline CODE_OF_CONDUCT.md:42]: Do we intend to keep the same code of conduct? Not blocking this PR though.

Comment (cartersocha) [reply CODE_OF_CONDUCT.md:42]: I think we'd remove it as part of the google, istio etc. clean up. Then use our [contributing markdown file](https://github.com/open-telemetry/opentelemetry-python/blob/main/CONTRIBUTING.md#contributing-to-opentelemetry-python) as the "code of conduct" equivalent. I looked through other repos & didn't notice a code of conduct specifically

File: docs/gcp-instrumentation.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # Integrating Online Boutique with Google Cloud
```
Comment (julianocosta89) [inline docs/gcp-instrumentation.md:1]: This is actually not valid anymore.
When adding OTel to the project I haven't updated all the docs, just the main README.
The GCP instrumentation was totally removed from the project, the environment variables were cleaned up and the code was removed from each service.

This file can be safely removed.

File: docs/development-principles.md [hunk 3]
```diff
@@ -3,26 +3,27 @@
...
-services. The experience of running the application locally or on GCP should
-be close.
+services. The experience of running the application locally or on GCP should be
+close.
 
 For example:
+
 - OpenCensus prints the traces to stdout when it cannot connect to GCP.
 - Stackdriver Debugging tries connecting to GCP multiple times, eventually gives
   up.
```
Comment (julianocosta89) [inline docs/development-principles.md:25]: OpenCensus and Stackdriver were removed from the project.

File: docs/memorystore.md [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
+  in the same region and use the same network as your instance.
+- You cannot connect to a Memorystore (redis) instance from a GKE cluster
+  without VPC-native/IP aliasing enabled.
 
 ![Architecture diagram with Memorystore](./img/memorystore.png)
 
 ## Steps
- 
+
 1. Create a GKE cluster with VPC-native/IP aliasing enabled.
```
Comment (julianocosta89) [inline docs/memorystore.md:17]: Will we keep the GCP instructions within the project?
I know that the initial microservices demo is from GCP, but I though the idea behind the opentelemetry-demo-webstore was to be a totally (as much as possible) vendor agnostic project.

File: docs/service-mesh.md [hunk 5]
```diff
@@ -1,14 +1,18 @@
-# Deploying to an Istio-enabled cluster 
+# Deploying to an Istio-enabled cluster
 
-This repository provides an [`istio-manifests`](/istio-manifests) directory containing ingress resources (an Istio `Gateway` and `VirtualService`) needed to expose the app frontend running inside a Kubernetes cluster.
+This repository provides an [`istio-manifests`](/istio-manifests) directory
+containing ingress resources (an Istio `Gateway` and `VirtualService`) needed to
+expose the app frontend running inside a Kubernetes cluster.
```
Comment (julianocosta89) [inline docs/service-mesh.md:5]: I haven't used Istio during the time I did the changes. 
The project is able to run with or without it.
Question is, do we wanna keep it?
No OTel is configured on it at the moment.

File: docs/service-mesh.md [hunk 6]
```diff
@@ -17,40 +21,44 @@ gcloud container clusters create onlineboutique \
...
+## Additional service mesh demos using OnlineBoutique
 
+- [Canary
+  deployment](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-canary-gke)
+- [Security (mTLS, JWT,
+  Authorization)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/security-intro)
+- [Cloud Operations
+  (Stackdriver)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-stackdriver)
+- [Stackdriver metrics (Open source
+  Istio)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/stackdriver-metrics)
```
Comment (julianocosta89) [inline docs/service-mesh.md:64]: If we decide to keep Istio, we should maybe take a look at this:
https://www.aspecto.io/blog/opentelemetry-and-istio-everything-you-need-to-know/

I have never tried myself, but it looks pretty easy to configure.

**Comment by cartersocha** [comment]
FYI @julianocosta89

File: docs/workload-identity.md [hunk 7]
```diff
@@ -1,14 +1,25 @@
-# Setup for Workload Identity clusters
```
Comment (julianocosta89) [inline docs/workload-identity.md:1]: As mentioned before, I don't think we should worry about cloud vendor specific stuff within this project.
My suggestion is to totally remove this file from this repo.

File: README.md [hunk 8]
```diff
@@ -80,13 +93,13 @@ ad06d33dea6d   redis:alpine                    "docker-entrypoint.s…"   42 sec
 eb39e7c3ff04   jaegertracing/all-in-one:1.30   "/go/bin/all-in-one-…"   43 seconds ago   Up 41 seconds   0.0.0.0:5775->5775/udp, :::5775->5775/udp, 0.0.0.0:5778->5778/tcp, :::5778->5778/tcp, 0.0.0.0:9411->9411/tcp, :::9411->9411/tcp, 0.0.0.0:14250->14250/tcp, :::14250->14250/tcp, 0.0.0.0:14268-14269->14268-14269/tcp, :::14268-14269->14268-14269/tcp, 0.0.0.0:6831-6832->6831-6832/udp, :::6831-6832->6831-6832/udp, 0.0.0.0:16686->16686/tcp, :::16686->16686/tcp   jaeger
 ```
 
-4. **Access the web frontend in a browser**:
```
Comment (julianocosta89) [inline README.md:83]: With the change all the steps lost their step number and all became `1.`.
Was that intentional?

File: docs/cloudshell-tutorial.md [hunk 9]
```diff
@@ -1,100 +1,132 @@
 # Online Boutique
```
Comment (julianocosta89) [inline docs/cloudshell-tutorial.md:1]: Should we keep the GKE instructions in here?

**Review by julianocosta89 — CHANGES_REQUESTED** [review]
Most of my points were regarding leaving GCP stuff within the docs.
I thought the initial cleanup was to also remove all vendor related stuff that was left within the repo.

Did I get it wrong?

**Review by julianocosta89 — APPROVED** [review]
As this PR has just markdown adjusts and nothing was removed, we are good to go!
🚀

**Pull request merged**